docs/software/integrations/mqtt/mosquitto.mdx

```diff
@@ -47,4 +47,54 @@ sidebar_position: 1
    brew services restart mosquitto
    ```
 
-7. If you are using the mac firewall, you will need to go into: System Preferences > Security & Privacy > Firewall > Firewall Options and add it.
\ No newline at end of file
+7. If you are using the mac firewall, you will need to go into: System Preferences > Security & Privacy > Firewall > Firewall Options and add it.
+
+
+### Using mosquitto on raspberry pi bookworm
+
+1. install mqtt server
+
+   ```sh
+   sudo apt-get install mosquitto mosquitto-clients
+   ```
+
+2. start the mqtt server
+
+   ```sh
+   sudo service mosquitto start
+   ```
+
+3. Do a quick test of server, start a subscriber on a topic:
+
+   Note: this will wait until you press control-c (publish a message, see below)
+
+   ```sh
+   mosquitto_sub -t test/hello
+   ```
+
+4. In another console, publish a message to that topic:
+
+   ```sh
+   mosquitto_pub -h localhost -q 0 -t test/hello -m 'yo!'
+   ```
+
+5. For Meshtastic to be able to access that server, two settings need to be changed in the
+   `/etc/mosquitto/conf.d/mosquitto.conf` file:
```

Comment:
in my install this was `/etc/mosquitto/mosquitto.conf'

Reply:
there was a note in one of the files to use the file I used here, cat the file was that the one?

Reply:
I've recently installed mosquitto on a pi and have no `/etc/mosquitto/conf.d/` folder.  Got a link to that note?